normandy/settings.py

```diff
@@ -15,6 +15,7 @@ class Core(Configuration):
         'normandy.selfrepair',
 
         'adminsortable',
+        'rest_framework',
```

Comment:
Oops, thanks. The docs for django-storages say you should add `'storages'` here too. http://django-storages.readthedocs.org/en/latest/#installation